web/src/features/Dashboard/components/Dashboard.tsx

```diff
@@ -170,6 +168,7 @@ const Dashboard = (props: Props) => {
   const history = useHistory();
   const match = useRouteMatch();
   const { app, isBundleUploading, isVeleroInstalled } = props;
+  const airgapUploader = useRef<AirgapUploader | null>(null);
```

Comment:
rather than storing the uploader on the state, we need to store it in a ref. async functions work differently in functional components and keep a reference to the context they're assigned in so the old `state.airgapUploader` references in async functions would never reference the latest version of state. 

https://css-tricks.com/dealing-with-stale-props-and-states-in-reacts-functional-components/

Reply:
There's probably a better way to do this but this is a quick fix to get the regression tests passing.